pytorch_lightning/trainer/deprecated_api.py

```diff
@@ -6,138 +6,190 @@
 
 
 class TrainerDeprecatedAPITillVer0_8(ABC):
-
     def __init__(self):
         super().__init__()  # mixin calls super too
 
     @property
     def nb_gpu_nodes(self):
         """Back compatibility, will be removed in v0.8.0"""
-        rank_zero_warn("Attribute `nb_gpu_nodes` has renamed to `num_nodes` since v0.5.0"
-                       " and this method will be removed in v0.8.0", DeprecationWarning)
+        rank_zero_warn(
+            "Attribute `nb_gpu_nodes` has renamed to `num_nodes` since v0.5.0"
+            " and this method will be removed in v0.8.0",
+            DeprecationWarning,
+        )
         return self.num_nodes
 
     @property
     def num_gpu_nodes(self):
         """Back compatibility, will be removed in v0.8.0"""
-        rank_zero_warn("Attribute `num_gpu_nodes` has renamed to `num_nodes` since v0.5.0"
-                       " and this method will be removed in v0.8.0", DeprecationWarning)
+        rank_zero_warn(
+            "Attribute `num_gpu_nodes` has renamed to `num_nodes` since v0.5.0"
+            " and this method will be removed in v0.8.0",
+            DeprecationWarning,
+        )
         return self.num_nodes
 
     @num_gpu_nodes.setter
     def num_gpu_nodes(self, num_nodes):
         """Back compatibility, will be removed in v0.8.0"""
-        rank_zero_warn("Attribute `num_gpu_nodes` has renamed to `num_nodes` since v0.5.0"
-                       " and this method will be removed in v0.8.0", DeprecationWarning)
+        rank_zero_warn(
+            "Attribute `num_gpu_nodes` has renamed to `num_nodes` since v0.5.0"
+            " and this method will be removed in v0.8.0",
+            DeprecationWarning,
+        )
         self.num_nodes = num_nodes
 
     @property
     def gradient_clip(self):
         """Back compatibility, will be removed in v0.8.0"""
-        rank_zero_warn("Attribute `gradient_clip` has renamed to `gradient_clip_val` since v0.5.0"
-                       " and this method will be removed in v0.8.0", DeprecationWarning)
+        rank_zero_warn(
+            "Attribute `gradient_clip` has renamed to `gradient_clip_val` since v0.5.0"
+            " and this method will be removed in v0.8.0",
+            DeprecationWarning,
+        )
         return self.gradient_clip_val
 
     @gradient_clip.setter
     def gradient_clip(self, gradient_clip):
         """Back compatibility, will be removed in v0.8.0"""
-        rank_zero_warn("Attribute `gradient_clip` has renamed to `gradient_clip_val` since v0.5.0"
-                       " and this method will be removed in v0.8.0", DeprecationWarning)
+        rank_zero_warn(
+            "Attribute `gradient_clip` has renamed to `gradient_clip_val` since v0.5.0"
+            " and this method will be removed in v0.8.0",
```

Comment:
```suggestion
            "Attribute `gradient_clip` has renamed to `gradient_clip_val` since v0.5.0 and this method will be removed in v0.8.0",
```